### ISSUE TITLE
Not compatible with mongoid 7.1

### ISSUE THREAD

**Issue by ynagorny** [PerfectMemory/mongoid-multitenancy#30]
Bundler reports that the mongoid-multitenancy version 2.0.2 is not compatible with mongoid 7.1:

```
Bundler could not find compatible versions for gem "mongoid":
  In Gemfile:
    mongoid (~> 7.1)

    mongoid-multitenancy (~> 2.0.2) was resolved to 2.0.2, which depends on
      mongoid (< 8, ~> 6)
```

**Comment by Richard-Degenne** [comment]
Oh, yeah. `~> 6` is equivalent to `~> 6.0`, which explains why mongoid 7 is not compatible.

Thanks for the report.

cc @abrisse

**Comment by abrisse** [comment]
Resolved in [2.0.3](https://github.com/PerfectMemory/mongoid-multitenancy/releases/tag/v2.0.3)
